custom_components/tuya_local/devices/trotec_pac-w_heatpump.yaml

```diff
@@ -0,0 +1,80 @@
+name: Air conditioner
+products:
+  - id: 6r9as02gkiaaxelt
+    name: Trotec PAC-W 2600 SH
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: COOL
+              value: cool
+            - dps_val: DRY
+              value: dry
+            - dps_val: FAN
+              value: fan_only
+            - dps_val: HEAT
+              value: heat
+    - id: 2
+      type: integer
+      name: temperature
+      range:
+        min: 16
+        max: 30
+      unit: C
+    - id: 3
+      type: integer
+      name: current_temperature
+    - id: 4
+      type: string
+      name: mode
+      hidden: true
+    - id: 5
+      name: fan_mode
+      type: integer
```

Comment:
Is this really an integer and not a string that happens to contain integer values?

Reply:
In API Explorer it is represented as a string, but it also works as an integer.